app/controllers/user/birth_records_controller.rb

```diff
@@ -26,10 +26,8 @@ def query
   # Attempts to add a record which is already attached will be ignored (by
   # 'distinct' modifier on User.birth_records).
   def add
-    AuditedOperationsService.add_birth_record_to_user(
-      user: current_user,
-      birth_record: BirthRecord.find_by(params.permit(:id))
-    )
+    @birth_record = BirthRecord.find_by(params.permit(:id))
```

Comment:
```suggestion
    @birth_record = BirthRecord.find_by(params.permit(:id))
    authorize @birth_record
```

Reply:
This also won't work before #172

Reply:
This one should work tho

Reply:
Nah just checked it - doesn't work

Reply:
<img width="1208" alt="Screen Shot 2019-09-02 at 10 36 14 AM" src="https://user-images.githubusercontent.com/1615322/64083086-910a2000-cd6d-11e9-854e-af9703661ab1.png">